pkg/restore/restore.go

```diff
@@ -1085,6 +1120,18 @@ func (ctx *context) restoreItem(obj *unstructured.Unstructured, groupResource sc
 		restorePodVolumeBackups(ctx, createdObj, originalNamespace)
 	}
 
+	// Wait for a CRD to be available for instantiating resources
+	// before continuing.
+	if groupResource == kuberesource.CustomResourceDefinitions {
+		available, err := ctx.crdAvailable(name, resourceClient)
+		if err != nil {
+			addToResult(&errs, namespace, errors.Wrapf(err, "error verifying custom resource definition is ready to use"))
+		}
+		if !available {
```

Comment:
in the case where `crdAvailable` returns false + a non-nil error, we'll end up adding two errors to the result here - I think we should probably just log the error, WDYT?

Reply:
I'd still like to communicate to users if, somehow, the CRD wasn't ready but there were no errors, so I made that statement an else if rather than a plain if.